### DISCUSSION TITLE
Several reactants

### DISCUSSION THREAD

**Discussion by ChumakAndy** [abrechts/Phoenix-ELN#11] [General]
Is it possible to add multiple reactants, or do they all have to be added manually?

![test](https://github.com/abrechts/Phoenix-ELN/assets/74661134/90ffabb2-d51f-425b-9ebb-7016a35c7ca8)

**Comment by abrechts** [comment]
Thanks for your question. You have to do this manually, this is by design - please see the Wiki entry [Reaction Sketch](https://github.com/abrechts/Phoenix-ELN/wiki#reaction-sketch). The background is, believe it or not, that in the predecessor application _Espresso ELN_ many users also added all of their _reagents_ to the reaction sketch as additional reactants, possibly since they were not aware on how to do this from the protocol directly. 

Please note that after entering an additional reactant as reagent manually, it is remembered by the application next time by the name you assigned to it, along with molecular weight and density (if specified).

**Reply by ChumakAndy** [reply]
Thanks for the explanation! In many cases, it is really convenient to use already stored reagents.
